examples/shopping_ads/approve_merchant_center_link.py

```diff
@@ -0,0 +1,156 @@
+            "found with the following details:"
+        )
+
+        merchant_center_link_status_enum = client.get_type(
+            "MerchantCenterLinkStatusEnum", version="v5"
+        )
+
+        # Iterate through the results and filter for links with pending states.
+        for merchant_center_link in response.merchant_center_links:
+            print(
+                f"Link '{merchant_center_link.resource_name}' has status "
+                f"'{merchant_center_link.status}'."
+            )
+
+            if (merchant_center_link.status ==
+                    merchant_center_link_status_enum.PENDING and
+                    str(merchant_center_link.id) == merchant_center_account_id
+               ):
+                _update_merchant_center_link_status(
+                    client,
+                    customer_id,
+                    merchant_center_link_service,
+                    merchant_center_link,
+                    merchant_center_link_status_enum.ENABLED,
+                )
+    except GoogleAdsException as ex:
+        print(
+            f'Request with ID "{ex.request_id}" failed with status'
+            f'"{ex.error.code().name}" and includes the following errors:'
+        )
+        for error in ex.failure.errors:
+            print(f'\tError with message "{error.message}".')
+            if error.location:
+                for field_path_element in error.location.field_path_elements:
+                    print(f"\t\tOn field: {field_path_element.field_name}")
+        sys.exit(1)
+
+
+def _update_merchant_center_link_status(
+        client,
+        customer_id,
+        merchant_center_link_service,
+        merchant_center_link,
+        status,
+    ):
+    """Updates the status of a Merchant Center link request.
+
+    Args:
+        client: An initialized GoogleAdsClient instance.
+        customer_id: The client customer ID string.
+        merchant_center_link_service: A merchant center link service instance.
+        merchant_center_link: The merchant center link to be modified.
+        status: The updated status to apply to the merchant center link.
+    """
+    # Creates an operation.
+    operation = client.get_type("MerchantCenterLinkOperation", version="v5")
+    link_to_update = operation.update
+    link_to_update.resource_name = merchant_center_link.resource_name
+    # Enables the pending link.
+    link_to_update.status = status
+
+    field_mask = protobuf_helpers.field_mask(None, link_to_update)
+    operation.update_mask.CopyFrom(field_mask)
+
+    # Updates the link.
+    mutate_response = merchant_center_link_service.mutate_merchant_center_link(
+        customer_id, operation
+    )
+
+    # Displays the result.
+    print(
+        "The status of Merchant Center Link with resource name "
+        + f"'{mutate_response.result.resource_name}' to Google Ads account : "
+        + f"{customer_id} was updated to {status}."
```

Comment:
ditto prior comment: Print the status as string instead of the raw enum value

Reply:
Fixed.